### ISSUE TITLE
Python docker image is broken

### ISSUE THREAD

**Issue by nokome** [stencila/stencila#75]
See build status at https://hub.docker.com/r/stencila/ubuntu-14.04-python-2.7/builds/

**Comment by Backenkoehler** [comment]
Python Image is still broken, but with another error:
ImportError: Cairo backend requires that cairocffi or pycairo is installed.

Do you have any updates?

**Comment by nokome** [comment]
Hey @Backenkoehler, thanks for submitting this!

Actually, I was just debugging something in the hub's error logs - couldn't work it out, then saw this issue and it clicked : the Python docker image needs updating! It doesn't have a `stencila-session` script that is expected (the R one one is here: https://github.com/stencila/stencila/blob/master/docker/stencila-session.r). So when someone tries to launch an Python session it always fails. Sorry, been focussed on getting things running for R.

Beyond that, as you say, the Python image build is still broken. The simplest approach to fixing it is probably to change `FROM stencila/ubuntu-14.04` (which has a bunch of stuff that isn't really needed) to `FROM ubuntu:14.04` as in the R image. Then write a `stencila-session.py` and follow what is in the Dockerfile for the R image.

I don't think it would take too much work, but won't be able to do it for at least a couple of days. In the meantime, happy review a PR!

**Comment by Backenkoehler** [comment]
Hey @nokome,

tried to start stencila through your hub. It didn't worked (500 error on ajax request), so I tried the docker image. Maybe I'll take a look if you have some guidance for me? :)

Stephan

**Comment by nokome** [comment]
@Backenkoehler : I have fixed the Python image with commits bdd7b44ec89b82c4d160c9b5a0c01d3827642fc4 and 32c77ce

It is available via the docker hub as stencila/ubuntu-14.04-py-2.7 (note slight name change to be consistent with other usage of `py` for Python in Stencila).

I have also fixed some other Python sheet stuff. To try it out:

```
docker run --detach --publish=7373:7373 stencila/ubuntu-14.04-py-2.7 stencila-session
```

Then open `http://localhost:7373/demo/sheets/simple-py/` in your browser (this will clone that demo Python sheet into the Docker container and serve it up). Importing of modules (e.g. matplotlib) does not seem to work but that is related to the implementation of the Python execution context of the Docker image per se.

So closing this issue. Thanks for your input.
